web/explorer/src/components/DescriptionPanel.vue

```diff
@@ -32,6 +50,12 @@
                 >.
             </p>
         </div>
+        <div>
+            <p class="text-base p-3 max-w-75rem" style="max-width: 75ch">
```

Comment:
```suggestion
            <p class="text-base p-3 max-w-75rem">
```
remove this so sentence can take entire line?